app/assets/javascripts/questions.js

```diff
@@ -65,6 +65,27 @@ $('#question_display_Modal').on('show.bs.modal', function (event) {
 })
 
 
+// Creates the javscript for the Social modal that allows it to be populated with the data recieved upon clicking the modal link
+$('#social_display_Modal').on('show.bs.modal', function (event) {
+  var social_link = $(event.relatedTarget) // element that triggered the modal. social_link is populated with the data from the click 
```

Comment:
Generally statements in Javascript should be terminated with a semicolon: `;` -- you can add that in for all the statements in this event handler.